modules/gapi/samples/oak_small_hetero_pipeline.cpp

```diff
@@ -0,0 +1,61 @@
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/cpu/core.hpp>
+#include <opencv2/gapi/gframe.hpp>
+#include <opencv2/gapi/media.hpp>
+
+#include <opencv2/gapi/oak/oak.hpp>
+#include <opencv2/gapi/streaming/format.hpp> // BGR accessor
+
+#include <opencv2/highgui.hpp> // CommandLineParser
+
+const std::string keys =
+    "{ h help  |              | Print this help message }"
+    "{ output  | output.png   | Path to the output file }";
+
+#ifdef WITH_OAK_BACKEND
+
+int main(int argc, char *argv[]) {
+    cv::CommandLineParser cmd(argc, argv, keys);
+    if (cmd.has("help")) {
+        cmd.printMessage();
+        return 0;
+    }
+
+    const std::string output_name = cmd.get<std::string>("output");
+
+    // Heterogeneous pipeline:
+    // OAK camera -> Sobel -> streaming accessor (CPU)
+    cv::GFrame in;
+    cv::GFrame sobel = cv::gapi::oak::sobelXY(in);
+    // Default camera and then sobel work only with nv12 format
+    cv::GMat out = cv::gapi::streaming::Y(sobel);
+
+    auto args = cv::compile_args(cv::gapi::oak::ColorCameraParams{},
+                                 cv::gapi::combine(cv::gapi::oak::kernels(),
+                                                   cv::gapi::streaming::kernels(),
+                                                   cv::gapi::core::cpu::kernels()));
```

Comment:
you shouldn't pass these two, right? if the sample doesn't work without it, we should raise a bug.

Reply:
It works, removed the kernels